Source/ASNetworkImageNode.mm

```diff
@@ -888,7 +888,7 @@ - (void)_lazilyLoadImageIfNecessary
             if (delegateDidLoadImageFromCache) {
               [delegate imageNodeDidLoadImageFromCache:self];
             }
-            as_log_verbose(ASImageLoadingLog(), "Decached image for %@ img: %@ url: %@ cacheType: %@", self, [imageContainer asdk_image], URL, cacheType);
+            as_log_verbose(ASImageLoadingLog(), "Decached image for %@ img: %@ url: %@ cacheType: %ld", self, [imageContainer asdk_image], URL, (long) cacheType);
```

Comment:
`sizeof(NSInteger) > sizeof(long)`

Another way to do this is to keep the formatter the same but box the scalar ie `@(cacheType)`.